apps/editing-toolkit/editing-toolkit-plugin/error-reporting/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Wordpress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * External dependences
+ */
+import { throttle } from 'lodash';
+
+/**
+ * Module variables
+ */
+
+/**
+ * Interval for error reports so we don't flood te endpoint.
+ *
+ * @type {number} throttling interval (since the last request) in milliseconds.
+ */
+const REPORT_INTERVAL = 60000; //1 minute
+
+window.addEventListener(
+	'error',
+	throttle( ( { message, filename, lineno, colno } ) => {
+		const error = {
+			message,
+			//file: `${ filename }:${ lineno }:${ colno })`,
```

Comment:
I tried setting those attrs, but they seem to somehow cause the report to not be fetched/show in Kibana.

Reply:
In https://github.com/Automattic/wp-calypso/pull/49485/commits/e81d448dfad9c3be68520512f749fcd13c9f2117 I used error's `stack` to be passed with the `trace` object. It should cover all the info we need from that error, so I also removed the custom strings containing `filename`, `lineno` & `colno` info. Here's how it looks in Logstash now: 

![image](https://user-images.githubusercontent.com/1451471/107049400-5611e480-67ca-11eb-833b-23298df9621f.png)

What do you think? cc @sirreal